libs/SalesforceSwiftSDK/SalesforceSwiftSDK.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,840 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 48;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		B7DA7AC51FCDE08F0098C1EA /* SalesforceSwiftSDK.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7ABB1FCDE08E0098C1EA /* SalesforceSwiftSDK.framework */; };
+		B7DA7ACA1FCDE08F0098C1EA /* SalesforceSwiftSDKTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = B7DA7AC91FCDE08F0098C1EA /* SalesforceSwiftSDKTests.swift */; };
+		B7DA7ACC1FCDE08F0098C1EA /* SalesforceSwiftSDK.h in Headers */ = {isa = PBXBuildFile; fileRef = B7DA7ABE1FCDE08E0098C1EA /* SalesforceSwiftSDK.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		B7DA7AD71FCDE1CD0098C1EA /* libxml2.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7AD61FCDE1B70098C1EA /* libxml2.tbd */; };
+		B7DA7AD91FCDE1D80098C1EA /* libz.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7AD81FCDE1D80098C1EA /* libz.tbd */; };
+		B7DA7ADA1FCDE1E00098C1EA /* SalesforceAnalytics.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7ADB1FCDE1E00098C1EA /* SalesforceAnalytics.framework */; };
+		B7DA7ADC1FCDE1E70098C1EA /* SalesforceSDKCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7ADD1FCDE1E70098C1EA /* SalesforceSDKCore.framework */; };
+		B7DA7ADE1FCDE1F40098C1EA /* SmartStore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7ADF1FCDE1F40098C1EA /* SmartStore.framework */; };
+		B7DA7AE01FCDE1FA0098C1EA /* SmartSync.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7AE11FCDE1FA0098C1EA /* SmartSync.framework */; };
+		B7DA7AF61FCDE3900098C1EA /* SDKCoreExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = B7DA7AF51FCDE3900098C1EA /* SDKCoreExtensions.swift */; };
+		B7DA7AFE1FCDE6330098C1EA /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = B7DA7AFD1FCDE6330098C1EA /* AppDelegate.swift */; };
+		B7DA7B001FCDE6330098C1EA /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = B7DA7AFF1FCDE6330098C1EA /* ViewController.swift */; };
+		B7DA7B031FCDE6330098C1EA /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = B7DA7B011FCDE6330098C1EA /* Main.storyboard */; };
+		B7DA7B051FCDE6330098C1EA /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = B7DA7B041FCDE6330098C1EA /* Assets.xcassets */; };
+		B7DA7B081FCDE6330098C1EA /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = B7DA7B061FCDE6330098C1EA /* LaunchScreen.storyboard */; };
+		B7DA7B541FCDECCC0098C1EA /* PromiseKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7DA7B491FCDECA70098C1EA /* PromiseKit.framework */; };
+		B7DA7B6E1FCF043A0098C1EA /* TestExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = B7DA7B6D1FCF043A0098C1EA /* TestExtensions.swift */; };
+		B7FDBFF11FCF6D4600756FEB /* Logger.swift in Sources */ = {isa = PBXBuildFile; fileRef = B7FDBFF01FCF6D4600756FEB /* Logger.swift */; };
```

Comment:
Where is this file? I don't see it in the pull request. Also, we should rename it to `SalesforceSwiftLogger.swift` following the pattern of component-based logger names.

Reply:
@bhariharan  I will add it in the subsequent PR's. I will remove it from here for now. I can name it SalesforceSwiftLogger instead. Also begs the question of prefixing classes now in Swift. Swift classes are generally name-spaced to the module. We may have to do it just for the sake of maintaining consistency?